### PR TITLE
Development/alex/sgw 78 add basic nft support

### PR DESCRIPTION
**Pull request by AlexBrandes**
- Adds support for NFTs in main home scene. NFTs don't have a detail page and just link out to their OpenSea link. 
- Adds "nfts" state key that contains the list of wallet NFTs fetched from OpenSea
- Refactors AccountController logic for clarity

### CODE REVIEW

File: source/components/AssetItem/AssetItem.tsx [hunk 1]
```diff
@@ -17,13 +18,18 @@ import TextV3 from 'components/TextV3';
 ///////////////////////
 
 import { RootState } from 'state/store';
+import { AssetType } from 'state/vault/types';
```
Comment (cmirandac665) [inline source/components/AssetItem/AssetItem.tsx:21]: Can you please move this line down to the types section.

File: source/scripts/Background/controllers/WalletController.ts [hunk 2]
```diff
@@ -121,6 +121,7 @@ export class WalletController implements IWalletController {
   }
 
   async switchWallet(id: string) {
+    console.log('Switching wallet to: ', id);
```
Comment (cmirandac665) [inline source/scripts/Background/controllers/WalletController.ts:124]: Please remove this console log?

File: source/scripts/Background/migration/v3_3.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import { browser } from 'webextension-polyfill-ts';
```
Comment (cmirandac665) [inline source/scripts/Background/migration/v3_3.ts:1]: It looks like we will need to run through the upgrade path to make sure that the state shape migrated properly.

File: source/selectors/walletsSelectors.ts [hunk 4]
```diff
@@ -2,85 +2,134 @@
...
-    return [...wallets];
+/**
+ * Returns NFT assets 
+ * NFTs are fetched for the active network only so no activeNetwork checks are needed
+ */
+ const selectNFTAssets = createSelector(
+  getActiveWallet,
+  (activeWallet: IWalletState): IAssetState[] => {
+    if (!activeWallet?.assets) {
+      console.log('no activeWallet.assets - no NFTs: ', activeWallet.assets);
```
Comment (cmirandac665) [inline source/selectors/walletsSelectors.ts:113]: Do we need this console log here?

File: source/state/vault/index.ts [hunk 5]
```diff
@@ -32,18 +24,15 @@ const initialState: IVaultState = {
     [KeyringNetwork.Constellation]: DAG_NETWORK.main.id,
     [KeyringNetwork.Ethereum]: ETH_NETWORK.mainnet.id,
   },
-  version: '2.1.1' ,
+  version: '2.1.1',
```
Comment (cmirandac665) [inline source/state/vault/index.ts:27]: Should this version number be bumped?

File: source/scripts/Background/migration/v3_3.ts (hunk 3 above)
Comment (AlexBrandes) [reply source/scripts/Background/migration/v3_3.ts:1]: Yeah, agree. It's just adding a new key so pretty low risk but still need to test it.

File: source/state/vault/index.ts (hunk 5 above)
Comment (AlexBrandes) [reply source/state/vault/index.ts:27]: I'm not actually sure how we're using that. Frank was the last one to update it.

**Pull request merged**